index.bs

```diff
@@ -2858,17 +2858,16 @@ the inputs to the algorithm needed to compute the set.
 An effective overload set is used, among other things, to determine whether there are ambiguities in the
 overloaded operations, constructors and callers specified on an interface.
 
-The elements of an effective overload set are tuples of the form
-&lt;|callable|, |type list|, |optionality list|&gt;.  If the effective overload
+The elements of an effective overload set are [=tuples=] of the form
+(|callable|, |type list|, |optionality list|).  If the effective overload
```

Comment:
It might be nice to `<dfn>` each of these components of the tuple, so that later references to them can refer back to them. You could then describe each briefly, e.g. in a bulleted list or similar.